lms/views/lti/deep_linking.py

```diff
@@ -84,16 +84,21 @@ def deep_linking_launch(context, request):
 
 
 class DeepLinkingFieldsRequestSchema(JSONPyramidRequestSchema):
-    deep_linking_settings = fields.Dict(required=False, allow_none=True)
     content_item_return_url = fields.Str(required=True)
     content = fields.Dict(required=True)
     group_set = fields.Str(required=False, allow_none=True)
     title = fields.Str(required=False, allow_none=True)
 
 
-@view_defaults(
-    request_method="POST", renderer="json", schema=DeepLinkingFieldsRequestSchema
-)
+class LTI11DeepLinkingFieldsRequestSchema(DeepLinkingFieldsRequestSchema):
```

Comment:
Each version has the same concept but different types (str vs dict).

We already have different endpoints for each version, makes sense to have different schemas.